src/AkeneoPimEnterpriseClient.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Akeneo\PimEnterprise;
+
+use Akeneo\Pim\AkeneoPimClient;
+use Akeneo\Pim\Api\AssociationTypeApiInterface;
+use Akeneo\Pim\Api\AttributeApiInterface;
+use Akeneo\Pim\Api\AttributeGroupApiInterface;
+use Akeneo\Pim\Api\AttributeOptionApiInterface;
+use Akeneo\Pim\Api\CategoryApiInterface;
+use Akeneo\Pim\Api\ChannelApiInterface;
+use Akeneo\Pim\Api\CurrencyApiInterface;
+use Akeneo\Pim\Api\FamilyApiInterface;
+use Akeneo\Pim\Api\LocaleApiInterface;
+use Akeneo\Pim\Api\MeasureFamilyApiInterface;
+use Akeneo\Pim\Api\MediaFileApiInterface;
+use Akeneo\Pim\Api\ProductApiInterface;
+use Akeneo\Pim\Security\Authentication;
+use Akeneo\PimEnterprise\Api\PublishedProductApiInterface;
+
+/**
+ * This class is the implementation of the client to use the Akeneo PIM ENTERPRISE API.
+ *
+ * @author    Olivier Soulet <olivier.soulet@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class AkeneoPimEnterpriseClient extends AkeneoPimClient implements AkeneoPimEnterpriseClientInterface
```

Comment:
:( sadly I have to extends the community client

Reply:
agree. it's really not a problem to do that